app/uk/gov/hmrc/merchandiseinbaggage/auth/AuthRedirects.scala

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
```

Comment:
Please add test for this, thanks. Can just grab [this](https://github.com/hmrc/bc-passengers-stride-frontend/blob/main/test/controllers/auth/AuthRedirectsSpec.scala).